src/components/NonProfit/ContactComponent.tsx

```diff
@@ -0,0 +1,170 @@
+import {
+  Tabs,
+  TabList,
+  TabPanels,
+  Tab,
+  TabPanel,
+  ChakraProvider,
+  Flex,
+  Button,
+  Modal,
+  ModalBody,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalCloseButton,
+  Icon,
+  Image,
+  FormControl,
+  FormLabel,
+  Input,
+  Textarea,
+  useDisclosure,
+} from "@chakra-ui/react";
+import chakraTheme from "@/styles/chakraTheme";
+import { useState, useEffect } from "react";
+export default function ContactComponent() {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const [firstName, setfirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [email, setEmail] = useState("");
+  const [message, setMessage] = useState("");
+  const [valid, setValid] = useState(false);
+  const colors = { true: "#2352A0", false: "#D9D9D9" };
+  useEffect(() => {
+    if (firstName !== "" && lastName !== "" && email !== "" && message !== "") {
+      setValid(true);
+    } else {
+      setValid(false);
+    }
+  }, [firstName, lastName, email, message]);
+  const onCloseFeatured = () => {
+    setfirstName("");
+    setLastName("");
+    setEmail("");
+    setMessage("");
+    onClose();
+  };
+
+  return (
+    <ChakraProvider theme={chakraTheme}>
```

Comment:
make sure the contact us form still has margins and doesn't overextend the tab size even when the screen size is smaller (rn the margins shrink to nothing)

Reply:
I added a margin to the div containing the contact component, but what do you mean by overextending the tab size? Do you mean like the size of the entire component? Should I have like a fixed max or min width?

Reply:
<img width="1067" alt="Screenshot 2024-03-25 at 12 58 05 PM" src="https://github.com/GTBitsOfGood/jennifer-anns-group/assets/72580357/a42d0619-b191-40f5-98dd-53b894171a9d">
right now if the screen is narrow it ends up looking like this, so maybe ensure the text boxes can shrink if the screen size is smaller? by overextending the tab size I mean it's going past the horizontal line under Contact Jennifer Ann's. Also, ik the design says Jennifer Ann's but could you change the placeholder first name to John and last name to Doe?

Reply:
Ah okay, yeah I can do that.